.github/workflows/release.yaml

```diff
@@ -0,0 +1,14 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+jobs:
+  release:
+    uses: boldlink/terraform-module-support/.github/workflows/release.yaml@1.0.0
```

Comment:
Is this the right version?

Reply:
Yes, we however need to update the template repo to main